centos-ci/bootstrap.sh

```diff
@@ -1,5 +1,17 @@
 #!/bin/bash
+export TZ=UTC
+alias print_date='date --rfc-2822'
+alias unixdate='date +%s'
+export BEGINING=$(unixdate)
```

Comment:
Typo: BEGINING -> BEGINNING

(alternatively, use `START_TIME` and `time_since_start`)